std/concurrency.d

```diff
@@ -1615,7 +1608,7 @@ class Generator(T) :
         else
         {
             static assert(0,
-                    "Fiber front is rvalue and thus cannot be moved when it defines a postblit.");
+                    "Fiber front is always rvalue and thus cannot be moved if it defines a postblit.");
```

Comment:
Template guard won't do since then postblitting fibers would not implement InputRange anymore. I can clarify wording trough.